src/amo/components/UserReview/index.js

```diff
@@ -34,10 +34,12 @@ function reviewBody({
   content,
   html,
   id,
+  showMoreCardChildId = '',
```

Comment:
mmm should we set it to `id` instead?

Reply:
Yeah, looking at it again, we don't really need `showMoreCardChildId` at all, and we can just always use `id`.